### ISSUE TITLE
Expected Lua number

### ISSUE THREAD

**Issue by max107** [nvim-neo-tree/neo-tree.nvim#12]
```
:NeoTreeReveal
```

```
E5108: Error executing lua ...t/neo-tree.nvim/lua/neo-tree/sources/filesystem/init.lua:54: Expected Lua number
stack traceback:
        [C]: in function 'nvim_buf_line_count'
        ...t/neo-tree.nvim/lua/neo-tree/sources/filesystem/init.lua:54: in function 'reveal_file'
        ...t/neo-tree.nvim/lua/neo-tree/sources/filesystem/init.lua:171: in function 'reveal_current_file'
        [string ":lua"]:1: in main chunk
```

**Comment by cseickel** [comment]
I am not able to recreate the issue. Can you provide more details about:

- the steps you take to get to that point? Do you have a file open or some dashboard open? Is this the first time you are opening NeoTree in this session?
- What version of nvim are you running?
- What operating system are you on?
- What other plugins do you have loaded? Do you have a dotfiles repo or can you provide a minimal config that recreates this issue?

Thanks for taking the time to fill out this bug report help out on this issue!

**Comment by max107** [comment]
> the steps you take to get to that point? 

Just open neovim: `nvim path/to/file` and run `NeoTreeReveal` or `nvim path/to/folder` and then open the file.

> Do you have a file open or some dashboard open?

file

> Is this the first time you are opening NeoTree in this session?

Yes.

> What version of nvim are you running?

0.6.1

> What operating system are you on?

Ubuntu 20.04 & mac bigsur

> What other plugins do you have loaded? Do you have a dotfiles repo or can you provide a minimal config that recreates this issue?

I have no public repo with my vim configuration at this moment. i will try create it tomorrow.

Thanks!

**Comment by max107** [comment]
Maybe i can send some logs for you? how i can grab it from neovim?

**Comment by cseickel** [comment]
I don't think there are any logs that will help. Can you tell me if it works after you have run `:NeoTreeShow` once?

I'll take a look and try to add more parameter checking and error handling in the code. Maybe I can fix it without actually recreating it.

**Comment by cseickel** [comment]
Hey @max107, I'm sorry I didn't realize this earlier, but I think I already fixed this bug in main. I might make another release tonight or tomorrow that includes the fix. If you'd like to try it out now, switch your branch to "main" and try it out.

**Comment by shaksiper** [comment]
Isn't this the reason: neovim/neovim#16745 ?

**Comment by cseickel** [comment]
Thank you @shaksiper ! I didn't realize my neovim wasn't up to date. Now I get the error, I'll fix it right away.

**Comment by cseickel** [comment]
What a mess! I have a real fix in **main** now if you can try it out. I think I will need to revert back to an earlier build of neovim during the workday because so many of my other plugins are broken as well. I'll switch back tonight and do more testing.

**Comment by cseickel** [comment]
These fixes have been released in v1.5
